tests/test_discriminated_union.py

```diff
@@ -1973,13 +1973,13 @@ class LoopState(BaseModel):
     class LeafState(BaseModel):
         state_type: Literal['leaf']
 
-    AnyState = Annotated[Union[NestedState, LoopState, LeafState], Field(..., discriminator='state_type')]
+    AnyState = Annotated[Union[NestedState, LoopState, LeafState], Field(discriminator='state_type')]
     adapter = TypeAdapter(AnyState)
 
     assert adapter.core_schema['schema']['type'] == 'tagged-union'
     for definition in adapter.core_schema['definitions']:
         if definition['schema']['model_name'] in ['NestedState', 'LoopState']:
-            assert definition['schema']['fields']['substate']['schema']['schema']['type'] == 'tagged-union'
+            assert definition['schema']['fields']['substate']['schema']['type'] == 'tagged-union'
```

Comment:
As a consequence of the fixed bug

Reply:
Thanks for explaining. I'm surprised this changed the schema - I guess there was a `with_default` schema here previously?

Reply:
yes exactly

Reply:
Seems like a bug fix.